database/seeders/CategorySeeder.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\Category;
+use Illuminate\Database\Seeder;
+
+class CategorySeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+
+    private $category;
+    public function __construct(Category $category)
+    {
+        $this->category = $category;
+    }
+    public function run()
+    {
+        $categories = [
```

Comment:
These are not random data. These are fixed and required, and we will add new one in this table.
I think I can't use Factory this time.

Reply:
noted, but please use `create` instead of `insert` so that you don't have to set `created_at` and `updated_at` to `now`. If changed to `create`, you can remove `created_at` and `updated_at` in the array.

Reply:
this comment also apply to other seeders.

Reply:
by the way, we can still use factory even if the data are fixed. factory is flexible